docs/environment-variables.md

```diff
@@ -0,0 +1,59 @@
+# Environment variables
+
+In development, environment variables are resolved from the .env file
+In production you are expected to set environment variables during deployment (e. g. using Vault)
+
+## List of environment variables
+
+Legend:
+
+- M - mandatory
+- O - optional (default value)
+
+### app
+
+- (M) `APP_ENV` - environment, in which app is running. Supported values: 'production' | 'development' | 'staging'
+- (M) `APP_BIND_ADDRESS` - address, on which server will be listening for HTTP(S) connections. e. g. '0.0.0.0'
+- (O) `APP_PORT` - port, on which server will be listening for HTTP(S) connections (3000)
+- (M) `LOG_LEVEL` - logs starting from which level should be emitted. Supported values: 'fatal' | 'error' | 'warn' | 'info' | 'debug' | 'trace' | 'silent'
+- (O) `APP_VERSION` - application version, exposed via healthcheck endpoint ('VERSION_NOT_SET')
+- (O) `GIT_COMMIT_SHA` - SHA of a last commit of the deployed version ('COMMIT_SHA_NOT_SET')
+
+- (M) `DATABASE_URL` - full DB connection URL. e. g. 'mysql://root:rootpass@localhost:3306/service_db'
+- (M) `JWT_PUBLIC_KEY` - full public key for JWT token validation, with newlines replaced with '||', e. g. '-----BEGIN PUBLIC KEY-----||MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAzi4k4ful8Q65RWbHvZwD||jKNfspb89typkUATf8KXlYcWp6ibUG9nKpYrig3jmlCdMvCm+S7kZedACshFyRmm||1ocaWjRIt/jJyzntxnMIgWetTedZXXzlFbparDMrdEMmsPbM7LrByCU57iKloZEl||BhOSQZk/JbJK1YpozTCxcs28YlpnTuMBaXvXddrQuNHo+HYhK53XlFXyiOBzmEFY||cBrVqptdjA3z7uNNd6A4IAfEkRYp4lZxZgwTPyjYZ1oXmhalvbr6OAs9ujLIZPSM||QoP1VoHLdOqrs7QTmi2rrNCfIcFkFp02N39TovMm9zZQJjQvFEJqIKe4db2457vr||uJ5qxkWmbBu+/tf6ytKfbiA433neLSvpfquPXbq3OLGzJ4H2YHiHa0ddfUCqdN49||t5nCPEMp6OTa5kXuwObf8yvHyoP8HgQQD+/sftHUIE/1sdQ6fzB/9L+smzp5SW/X||nI8NY0k1SH9MLlweGuXi6M1jS62kPWk4HTDQmiqUTImcG0XYRrVd5ISXPdfnVgnq||KKht+SUmkPrfaWMDc21FsXXmmVSRTjvBhA6Cy6PLPzGZaeA4TVkOZUkp1OvcyfiI||HixuZca1OASxGeUM8lcPi9my8TJCtw5ZR0M/uqVV/1o3U0nx+U5z54ulWN9leMLY||vgv+lGrqfFWRemajGXSm8L0CAwEAAQ==||-----END PUBLIC KEY-----'
+
+### redis
+
+- (M) `REDIS_HOST` - Redis DB host
+- (M) `REDIS_PORT` - Redis DB port
+- (M) `REDIS_USERNAME` - Redis DB username
+- (M) `REDIS_PASSWORD` - Redis DB password
+- (M) `REDIS_DB` - Redis DB database (number in 0-15 range)
+- (O) `REDIS_USE_TLS` - whether to use https connection
```

Comment:
🟢  I'd added available values here, because there might be a confusion between (true/false, 0/1, yes/no)